serverless/client/client.go

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package client is a client for the serverless log.
+// Package client provides client support for interacting with a serverless
+// log.
+//
+// See the /cmd/client package in this repo for an example of using this.
```

Comment:
```suggestion
// See the /serverless/cmd/client package in this repo for an example of using this.
```

?

Reply:
The `repo` in the comment is trillian-examples, so I just fixed the path.